dockerizing-notes.md

```diff
@@ -19,6 +20,7 @@
 
 ## TODO:
 
-* figure out what is needed for also developing `cg-style` (team usually uses `npm link`)
 * figure out why the shrinkwrap doesn't work -- looks like it has to do
```

Comment:
Oh yeah, I think you need to have this stuff in there:

```dockerfile
RUN apt-get update && apt-get install -y -q --no-install-recommends \		
        build-essential \			
        python \		
        python-dev \		
    && rm -rf /var/lib/apt/lists/*
```

Haven't tested -- you might be able to get rid of `build-essential`, but I'm pretty sure you need `python` and `python-dev`.